source/_components/light.osramlightify.markdown

```diff
@@ -27,13 +27,57 @@ host:
   required: true
   type: string
 allow_lightify_nodes:
-  description: (true/false) Edit this to stop homeassistant from importing the lightify lights.
+  description: (true/false) Should Home Assistant import individual lights?
```

Comment:
Would rather make something that indicates: if you do `true` then this happens and if you do `false` then that happens....

Reply:
Done.